.buildkite/pipeline.yml

```diff
@@ -241,7 +306,15 @@ steps:
                     - BUILDKITE_PULL_REQUEST_BASE_BRANCH=${BUILDKITE_PULL_REQUEST_BASE_BRANCH}
                     - GITHUB_PR_LABELS=${GITHUB_PR_LABELS}
 
+  - label: "Triggering Build for Xpack/Winlogbeat"
```

Comment:
should we change the name a bit to help us understand that these aren't PR triggers? Something like:

`Triggering merge commit Build for x-pack/winlogbeat` ? It might be useful when users look at the BK run view to understand what was is being run?